djrichtextfield/mixins.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import unicode_literals
+
+from django.utils import six
+from django.utils.module_loading import import_string
+
+from djrichtextfield import settings
+
+
+def pass_value(value):
+    return value
+
+
+class SanitizerMixin(object):
+    """
+    Get the field sanitizer from the provided kwargs during init,
+    or from the settings.
+    """
+
+    SANITIZER_KEY = 'sanitizer'
+    SANITIZER_PROFILES_KEY = 'sanitizer_profiles'
+
+    def __init__(self, *args, **kwargs):
+        self.sanitizer = kwargs.pop('sanitizer', None)
+        super(SanitizerMixin, self).__init__(*args, **kwargs)
+
+    def get_sanitizer(self):
+        """
+        Get the field sanitizer.
+
+        The priority is the first defined in the following order:
+        - A sanitizer provided to the widget.
+        - Profile (field settings) specific sanitizer, if defined in settings.
+        - Global sanitizer defined in settings.
+        - Simple no-op sanitizer which just returns the provided value.
+
+        """
```

Comment:
👍 indeed a bit more condensed/readable. Still needs a small tweak in case no sanitizer is configured for the chosen profile it should fallback to the global sanitizer.